core/src/test/java/com/sequenceiq/cloudbreak/service/cluster/ClusterCommonServiceTest.java

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
do we need this ? @oleewere

Reply:
@doktoric you mean the unit test or the whole feature? actually I had this in an another tool, so i discussed with Attila to ship here in java as an experimental feature, but then it was told to fine to put it into the stack endpoint

Reply:
ah ok, now from checkstyle error I see you referred to the license, yeah we do not need that one:)